CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ## [NEXT RELEASE]
 
 ### Added Features
+- The Central PVC stackrox-db is no longer required after this upgrade. To obsolete existing PVC, please check the docs online.
```

Comment:
Would this make more sense under `Technical Changes`?  Seems weird to have it under `Added Features` though that may be the correct place.

Reply:
I am also hesitating. i will make that change.